independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/AbstractInstanceHandle.java

```diff
@@ -18,7 +18,7 @@ abstract class AbstractInstanceHandle<T> implements InstanceHandle<T> {
     private static final AtomicIntegerFieldUpdater<AbstractInstanceHandle> DESTROYED_UPDATER = AtomicIntegerFieldUpdater
             .newUpdater(AbstractInstanceHandle.class, "destroyed");
 
-    private final InjectableBean<T> bean;
+    protected final InjectableBean<T> bean;
```

Comment:
This change is useless, right? (Just nitpicking.)

Reply:
Ah yes, I used it in `EagerInstanceHandle` in the first iteration. Do you want me to revert this change? It would take a while because I switched to another task already...

Reply:
No, feel free to leave it as is. I was just wondering.